index.html

```diff
@@ -10,234 +10,252 @@
-      Hello I'm a software developer! I can help you build a product ,
-      feature or website Look through some of my work and experience!
-      If you like what you see and have a project you need coded, don't
-      hestiate to contact me.
+      <ul>
+        <li>
+          <h1 class="topic item">Languages</h1>
+          <ul class="languages">
+            <li>
+              <img src="image/js.svg" alt="" />
+              JavaScript
+            </li>
+            <li>
+              <img src="image/html5.svg" alt="" />
+              HTML
+            </li>
+            <li>
+              <img src="image/css3.svg" alt="" />
+              CSS
+            </li>
+          </ul>
+        </li>
+        <li class="topic item">Frameworks</li>
+        <li class="topic item">Skills</li>
+      </ul>
     </div>
-    <div class="menuSection">
-      <a href="#" target="_blank">LET'S CONNECT</a>
-      <div class="socialMedia">
-        <img src="image/Twitter.png" alt="" />
-        <img src="image/Linkedin.png" alt="" />
-        <img src="image/Meduim.png" alt="" />
-        <img src="image/Github.png" alt="" />
-        <img src="image/angellist.png" alt="" />
+
+    <footer class="footer">
+      <div class="contact-me">
+        <h2>Contact me</h2>
+        <p>
+          If you have an application you are interested in developing, a
+          feature that you need built or a project that needs coding. I’d love
+          to help with it
+        </p>
       </div>
-    </div>
-    <div class="project">Get my resume</div>
-
-    <ul>
-      <li>
-        <div class="topic item">Languages</div>
-        <ul class="languages">
-          <li> <img src="image/js.svg" alt=""> JavaScript</li>
-          <li> <img src="image/html5.svg" alt=""> HTML</li>
-          <li> <img src="image/css3.svg" alt=""> CSS</li>
-        </ul>
-      </li>
-      <li class="topic item">Frameworks</li>
-      <li class="topic item">Skills</li>
-    </ul>
-  </div>
 
-  <footer class="footer">
-    <div class="contact-me">
-      <h2>Contact me</h2>
-      <p>
-        If you have an application you are interested in developing, a feature that you need built or a project that
-        needs coding. I’d love to help with it
-      </p>
-    </div>
+      <div class="contact-forms">
+        <form class="message-form" action="https://formspree.io/f/xqknrzed" method="post">
 
-    <div class="contact-forms">
-      <form class="message-form" action="https://formspree.io/f/xqknrzed" method="post">
+          <div class="input-li">
+            <input type="text" name="text" id="text-input" placeholder="Yeremias" maxlength="30" required>
+          </div>
+          <div class="input-li">
+            <input type="email" name="email" id="mail" placeholder="natajay14@gmail.com" required>
+          </div>
+          <div class="text-area-li">
+            <textarea name="msg" id="msg" maxlength="500" placeholder="Write your messages here" required></textarea>
```

Comment:
- [ ] Great job working here on the form As I can see here the background here in not fill all the action so kindly adjust it to match the Figma as much as possible and to fill all the white spaces as shown here.💪🏻


- Your design
![image](https://user-images.githubusercontent.com/58553711/179356707-44b35c73-4cb4-4d1c-b815-7ea4bafb46e4.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/179356721-06fa955a-3585-4c9f-ab96-5b95fc6fdc26.png)